test/caching.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+const CACHE_TESTS = [
+  'https://html.spec.whatwg.org/fonts/Essays1743.ttf',
```

Comment:
These tests will actually fail until HTML is hosted on marquee, so this should be merged around the same time.

Reply:
I fixed this instead.